Objective-C/Tests/DocumentTest.m

```diff
@@ -2292,8 +2320,9 @@ - (void) testDocumentRevisionHistory {
     Assert([defaultCollection saveDocument:doc error: &err]);
     Assert(doc._getRevisionHistory);
     
-    doc = [[defaultCollection documentWithID: @"doc1" error: &err] toMutable];
-    Assert(doc._getRevisionHistory);
+    // Fails from LiteCore,only 1 rev, no rev history
+    // doc = [[defaultCollection documentWithID: @"doc1" error: &err] toMutable];
+    // Assert(doc._getRevisionHistory);
```

Comment:
Is this expected or failed?

Reply:
one change slipped when I resolved conflicts... by default, we only get current rev from LiteCore for a doc, so that would've failed. Now we grab using correct content flag, only for the revHist functionality